charts/greptimedb-cluster/README.md

```diff
@@ -1,6 +1,6 @@
 # greptimedb-cluster
 
-A Helm chart for deploying GreptimeDB cluster in Kubernetes
```

Comment:
suggestion: we need change [README.md.gotmpl](https://github.com/GreptimeTeam/helm-charts/blob/main/charts/greptimedb-cluster/README.md.gotmpl) , and running `make update-docs` can generates README.md.